.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,165 @@
+name: Integration Tests
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+env:
+  TIMEOUT: 5m
+  TYK_OSS_NS: tyk-oss
+  TYK_MDCB_DATAPLANE_NS: tyk-mdcb-dataplane
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+        with:
+          version: v3.3.0
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config ct.yaml --all
+
+  integration-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.11.2
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          check-latest: true
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.4.0
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.4.0
+
+      # Tyk OSS tests
+      - name: Deploy Tyk OSS and Dependencies
+        run: |
+          kubectl create namespace ${{ env.TYK_OSS_NS }}
+          
+          # Do not change the name
+          helm install redis simple-redis --wait --repo 'https://helm.tyk.io/public/helm/charts/' -n ${{ env.TYK_OSS_NS }} --timeout ${{ env.TIMEOUT }}
+          
+          helm dependency update ./tyk-oss
+
+      - name: Run chart-testing (install)
+        run: ct install --namespace ${{ env.TYK_OSS_NS }} --charts ./tyk-oss --helm-extra-set-args "--wait=true"
```

Comment:
You have used `ct install` command to run tests here and `helm test` to run tests of `mdcb-data-plane`. 
Is it possible to use the same command just for consistency?

Reply:
oh okay